src/Sylius/Behat/Context/Ui/Admin/ManagingOrdersContext.php

```diff
@@ -707,9 +707,9 @@ public function theFirstOrderShouldHaveNumber($number)
     /**
      * @Then it should have shipment in state :shipmentState
      */
-    public function itShouldHaveShipmentState($shipmentState)
+    public function itShouldHaveShipmentState(string $shipmentState): void
     {
-        Assert::true($this->showPage->hasShipment($shipmentState));
+        Assert::true($this->showPage->hasShipmentWithStatus($shipmentState));
```

Comment:
WDYT about avoiding mixing `state` and `status` in naming?
```suggestion
        Assert::true($this->showPage->hasShipmentWithState($shipmentState));
```